<launch>
    <arg name="map_file" default="$(find course_agv_slam)/mapping/bayes.yaml"/>
    <!-- Run the map server -->
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />

	<!-- <node pkg="course_agv_slam" type="localization.py" name="icp" output="screen">	
		<param name="robot_x" value="0"/>
		<param name="robot_y" value="0"/>
		<param name="robot_theta" value="0"/>

		<param name="max_iter" value="10"/>
		<param name="tolerance" value="0"/>
		<param name="dis_th" value="3"/>
	</node> -->

	<node pkg="course_agv_slam" type="localization_lm.py" name="localization" output="screen" >
		<param name="range_threshold" value="0.5"/>
		<param name="radius_max_th" value="0.1"/>
		<param name="landMark_min_pt" value="2"/>
		<param name="min_match" value="1"/>
	</node>

	<node pkg="course_agv_slam" type="wheel.py" name="wheel_odometry" output="screen"/>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find course_agv_slam)/launch/course_agv.rviz" />
</launch>
